workloads/kube-burner/run_maxnamespaces_test_fromgit.sh

```diff
@@ -11,10 +11,31 @@ export REMOTE_METRIC_PROFILE=${REMOTE_METRIC_PROFILE:-https://raw.githubusercont
 
 deploy_operator
 check_running_benchmarks
+if [[ ${PPROF_COLLECTION} == "true" ]] ; then
+  delete_pprof_secrets
+  delete_oldpprof_folder
+  get_pprof_secrets
+fi 
 deploy_workload
 wait_for_benchmark ${WORKLOAD}
 rm -rf benchmark-operator
 if [[ ${CLEANUP_WHEN_FINISH} == "true" ]]; then
   cleanup
 fi
+delete_pprof_secrets
+
+if [[ ${ENABLE_SNAPPY_BACKUP} == "true" ]] ; then
+ echo -e "snappy server as backup enabled"
+ source ../../utils/snappy-move-results/common.sh
+ 
+ tar -zcvf pprof.tar.gz ./pprof-data
+ 
+ export workload=kube-burner-maxnamespaces
+
+ export snappy_path="$SNAPPY_USER_FOLDER/$runid$platform-$cluster_version-$network_type/$workload/$folder_date_time/"
+ generate_metadata > metadata.json  
+ ../../utils/snappy-move-results/run_snappy.sh pprof.tar.gz $snappy_path
+ ../../utils/snappy-move-results/run_snappy.sh metadata.json $snappy_path
+ store_on_elastic
+fi
```

Comment:
We can move all this logic to it's own function in common.sh and just leave something like below in the workload specific files

```shell
if [[ ${ENABLE_SNAPPY_BACKUP} == "true" ]] ; then
  snappy_backup kube-burner-maxnamespaces
fi
```

Reply:
yes i've been thinking about this as well, will do that